integration-test/src/features/exceed-the-limits-in-upload.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Exceed size limit in upload
+
+  Background:
+    Given GPD-Upload running
+
+  Scenario: Create Debt Position by file upload KO: file size > limit size
```

Comment:
add `limit size (xxxMb)`

Reply:
The effective size, greater than the limit, is already in the scenario, one day it could be GB, I prefer maintain scenario description more generic whether you agree